clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerNetworkThread.java

```diff
@@ -178,27 +171,11 @@ static void runAtClose(final Collection<Optional<? extends RequestManager>> requ
                            final NetworkClientDelegate networkClientDelegate,
                            final Timer timer) {
         // These are the optional outgoing requests at the
-        List<NetworkClientDelegate.PollResult> pollResults = requestManagers.stream()
+        requestManagers.stream()
```

Comment:
see the comment in the FetchRequestManager

Reply:
@kirktrue

Reply:
This is not a conflict actually - this is just some changes to how fetch request manager closes